pkg/gadgettracermanager/stream/stream.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stream
+
+import (
+	"sync"
+	"time"
+)
+
+const (
+	MAX_BUFFERED_LINES = 100
+)
+
+type TimestampedLine struct {
+	Line      string
+	Timestamp time.Time
```

Comment:
I'm wondering if you plan to use this field in the future as it isn't used in this PR.

Reply:
I was pondering adding a timestamp in the json output but I didn't do it...